.dagger/main.go

```diff
@@ -16,9 +16,10 @@ import (
 type DaggerDev struct {
 	Src *dagger.Directory // +private
 
-	Version string
-	Tag     string
-	Git     *dagger.VersionGit // +private
+	EngineVersion string
```

Comment:
This looks accidental? Not sure where this is set/used.

Reply:
yes it is accidental. I'll remove these.